src/main/java/com/acrolinx/sidebar/utils/OSUtils.java

```diff
@@ -6,44 +6,46 @@
  * Retrieves the current OS.
  */
 
-@SuppressWarnings("unused")
-public class OSUtils
+public final class OSUtils
 {
     public enum EnumOS
     {
         LINUX, MACOS, UNKNOWN, WINDOWS;
 
         public boolean isLinux()
         {
-
             return this == LINUX;
         }
 
         public boolean isMac()
         {
-
             return this == MACOS;
         }
 
         public boolean isWindows()
         {
-
             return this == WINDOWS;
         }
     }
 
     public static EnumOS getOS()
     {
-        String s = System.getProperty("os.name").toLowerCase();
+        String string = System.getProperty("os.name").toLowerCase();
 
-        if (s.contains("mac"))
+        if (string.contains("mac")) {
```

Comment:
Wouldn't _osName_ be a better variable name?

Reply:
I changed the variable name to "osNameString" in commit https://github.com/acrolinx/sidebar-sdk-java/pull/43/commits/4e5fb58b267e5d0a67afcb180cc725520e2c38aa